flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/catalog/AbstractJdbcCatalog.java

```diff
@@ -116,6 +122,36 @@ public String getBaseUrl() {
 		return baseUrl;
 	}
 
+	// ------ Postgres default objects that shouldn't be exposed to users ------
+
+	/**
+	 * Retrieve the list of system schemas to ignore.
+	 */
+	public abstract Set<String> getBuiltinSchemas();
+
+	/**
+	 * Retrieve the list of system database to ignore.
+	 */
+	public abstract Set<String> getBuiltinDatabases();
```

Comment:
I think we don't need to move this interfaces into base classes and `JdbcCatalog`, because they are not needed in `getPrimaryKey`.